docs/manual/scala/guide/cluster/MigratingToAkkaPersistenceTyped.md

```diff
@@ -84,7 +84,7 @@ That is, instead of injecting a `persistentEntityRegistry`, use a `clusterShardi
 
 ### Registration: caveats
 
-Even though there is no longer a `PersistentEntity` instance to register, the `persistentEntityRegistry` is still necessary to build `TopicProducer`'s. When writing a `Service` implementation that includes a [[Topic Implementation|MessageBrokerApi#Implementing-a-topic]] the `TopicProducer` API requires an `eventStream` that is provided by the `persistentEntityRegistry`. This means in some cases you can't get rid of the `persistentEntityRegistry` wyou were injecting in your `Service` implementation even if you use the `clusterSharding` to obtaing a reference to your persistent behavior.
+Even though there is no longer a `PersistentEntity` instance to register, the `persistentEntityRegistry` is still necessary to build `TopicProducer`'s. When writing a `Service` implementation that includes a [[Topic Implementation|MessageBrokerApi#Implementing-a-topic]] the `TopicProducer` API requires an `eventStream` that is provided by the `persistentEntityRegistry`. This means in some cases you can't get rid of the `persistentEntityRegistry` you were injecting in your `Service` implementation even if you use the `clusterSharding` to obtaing a reference to your persistent behavior.
```

Comment:
> to obtaing

🤔

Reply:
It turns out there are a few spelling errors in this doc. I'm adding another commit for the ones I found